src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+                try {
+                    String task = input.split(" ", 2)[1];
+                    inputs.add(new Todo(task, false));
+                    System.out.println(DECO_LINE);
+                    System.out.println(ADD_TASK);
+                    System.out.println("   [T][ ] " + task);
+                    System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                    System.out.println(DECO_LINE);
+                } catch (Exception nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong! You should write \"todo <event>\"");
+                    System.out.println(DECO_LINE);
+                }
+            } else if (input.split(" ")[0].equals("deadline")) {
+                try {
+                    String task = input.split(" /by ")[0].split(" ", 2)[1];
+                    String deadline = input.split(" /by ")[1];
+                    inputs.add(new Deadline(task, false, deadline));
+                    System.out.println(DECO_LINE);
+                    System.out.println(ADD_TASK);
+                    System.out.println("   [D][ ] " + task + " (by: " + deadline + ")");
+                    System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                    System.out.println(DECO_LINE);
+                } catch (Exception nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong! You should write \"deadline <event> /by <deadline>\"");
+                    System.out.println(DECO_LINE);
+                }
+            } else if (input.split(" ")[0].equals("event")) {
+                try {
+                    String task = input.split(" /at ")[0].split(" ", 2)[1];
+                    String schedule = input.split(" /at ")[1];
+                    inputs.add(new Event(task, false, schedule));
+                    System.out.println(DECO_LINE);
+                    System.out.println(ADD_TASK);
+                    System.out.println("   [E][ ] " + task + " (at: " + schedule + ")");
+                    System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                    System.out.println(DECO_LINE);
+                } catch (Exception nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong! You should write \"event <event> /at <schedule>\"");
+                    System.out.println(DECO_LINE);
+                }
+            } else {
+                System.out.println(DECO_LINE);
+                System.out.println(ERROR_MESSAGE);
+                System.out.println(DECO_LINE);
+            }
+            input = readinput.nextLine();
+        }
+
+        storage.saveTasks(inputs);
+        System.out.println(DECO_LINE);
+        System.out.println(BYE_MESSAGE);
+        System.out.println(DECO_LINE);
     }
 }
```

Comment:
Same as previous comments. This main method is a bit too long. Please consider using abstract methods to handle the operation.